packages/vue-client/src/utils/voice-synthesizer.ts

```diff
@@ -0,0 +1,8 @@
+// https://github.com/mdn/dom-examples/blob/main/web-speech-api/speak-easy-synthesis/script.js
+
+export function stop_and_speak(text: string): void {
+  window.speechSynthesis.cancel();
```

Comment:
I added this cancel to make sure that there is no delay for the next count-down.

But I'm not sure if this has any consequences for screen readers, like if they use the same queue for the text-to-speech.